adapters/coap-vertx-base/src/main/java/org/eclipse/hono/adapter/coap/AbstractVertxBasedCoapAdapter.java

```diff
@@ -1059,6 +1066,36 @@ private void addCommandReceptionTimer(
         context.put(KEY_TIMER_ID, timerId);
     }
 
+    /**
+     * Sets a timer to trigger the sending of a separate ACK to a device.
+     * <p>
+     *
+     * @param context The device's currently executing HTTP request.
+     * @param timeoutMillis The number of milliseconds to wait for a separate ACK.
+     */
+    private void addSeparateAcceptTimer(
```

Comment:
how about moving this method to CoapContext?

Reply:
Done.